packages/create-mechanic/function-templates/svgjs-image/index.js

```diff
@@ -0,0 +1,74 @@
+import { SVG, extend as SVGextend, Element as SVGElement } from '@svgdotjs/svg.js'
+
+export const handler = ({ inputs, mechanic }) => {
+  const { width, height, text, color1, color2, radiusPercentage } = inputs;
+
+  const center = [width / 2, height / 2];
+  const radius = Math.max(radiusPercentage / 100, 0.1);
+  const angle = Math.random() * 360;
+
+  let draw = SVG().size(width, height)
+  , rawSVG = '<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90" ><defs></defs><path class="cls-1" d="M90,45A45,45,0,0,0,0,45Z"/></svg>'
+  , arcGroup = draw.group()
+  , arc1 = arcGroup.group().svg(rawSVG)
+  , arc2 = arcGroup.group().svg(rawSVG)
```

Comment:
@munusshih Looks like this hasn't been addressed. Do you have any thoughts about it?

Reply:
> @fdoflorenzano yes! Did you see my above comments?

I didn't 😂 .

I see now, it's an interesting use case. In that case, I suggest two things:
- Move the `rawSvg` var definition outside of the function, after the imports. Since it's a constant value, it's a bit of a mouthful of a sentence in the middle of the code. 
- Before the `rawSvg` definition, I would add a comment that explains it's usage. Maybe something like "this was exported from ..." "It can be replaced with another path export!" Or something like that. So that it's clearer on it's intention.

Reply:
> I didn't 😂 . 
That's fair cause it was pending. 😂 I forgot to submit them.
The suggestions sound great! I will implement them now.